app/Http/Controllers/ViolenceTypeController.php

```diff
@@ -15,7 +15,7 @@ class ViolenceTypeController extends Controller
      */
     public function index()
     {
-        //
+        return ViolenceType::all();
```

Comment:
I think we should format our responses so that the client app knows what to expect.

Reply:
@offiahchiaz I'll make some updates

Reply:
Resolved by https://github.com/Policy-Lab-Africa/violenceTrackerApi/pull/2/commits/0e2dced52a9dd4c93e5696f07d0f4cdb86141b0d